docs/cluster-management/configure-api.md

```diff
@@ -242,42 +246,51 @@ Set these environment variables:
 
 | Environment name           | Required                  | Default Value | Description                                  |
 |:---------------------------|:--------------------------|:--------------|:---------------------------------------------|
-| SCM_PLUGIN                 | No                        | github        | `github` or `bitbucket`                      |
+| SCM_SETTINGS               | Yes                       |               | JSON object with SCM settings                |
 | SECRET_OAUTH_CLIENT_ID     | Yes                       |               | Your OAuth Client Id (Application key)       |
 | SECRET_OAUTH_CLIENT_SECRET | Yes                       |               | You OAuth Client secret (Application secret) |
 | WEBHOOK_GITHUB_SECRET      | Yes for Github            |               | Secret to sign for webhooks                  |
 | SCM_GITHUB_GHE_HOST        | Yes for Github Enterprise |               | GHE host for Github Enterprise               |
-| SCM_PRIVATE_REPO_SUPPORT   | No                        | false         | Ask Github users for 'repo' scope to allow read/write access to public and private repo |
-| SCM_USERNAME               | No                        | sd-buildbot   | Username for checkout                        |
-| SCM_EMAIL                  | No                        | dev-null@screwdriver.cd | Email of user for checkout         |
+| SCM_PRIVATE_REPO_SUPPORT   | Yes for Github            | false         | Ask Github users for 'repo' scope to allow read/write access to public and private repo |
+| SCM_USERNAME               | Yes for Github            | sd-buildbot   | Username for checkout                        |
```

Comment:
These are also not required: https://github.com/screwdriver-cd/scm-github/blob/master/index.js#L88-L89